### DISCUSSION TITLE
display search results as a custom tile look & feel with buttons etc

### DISCUSSION THREAD

**Discussion by sudeshw** [microsoft-search/pnp-modern-search#3045] [Q&A] [answered]
Can i use this search extensibility library to display results as a tile and have custom content layout within tile. need to disaply buttons, menu etc for each result tile. Button click should update same list and also should call APIs. 4 tiles should be displayed in a row.

**Comment by wobba** [comment, marked as answer]
Yes, a web component should work for this.

**Comment by sudeshw** [comment]
I deployed extensibility samples with pnp search results web part. How can i check the Web Component there?

**Reply by wobba** [reply]
Load the manifest id as per https://microsoft-search.github.io/pnp-modern-search/extensibility/#register-your-extensibility-library-with-a-web-part, and add the component to a template and see if it shows.
